lib/messageEvents.js

```diff
@@ -1,33 +1,22 @@
 'use strict';
 
-var api = 'message-events';
+const api = 'message-events';
```

Comment:
are we dropping node 0.12 support? I think I remember that discussion somewhere, but just wanted to make sure.